app/controllers/admin/cafe_works_controller.rb

```diff
@@ -0,0 +1,115 @@
+# encoding:UTF-8
+class Admin::CafeWorksController < ApplicationController
+  before_action :authenticate
+  before_action :set_cafe_work, only: [:edit, :show, :update, :destroy, :remove_worker]
+  before_action :new_cafe_work, only: [:new, :setup, :setup_create, :create]
+  before_action :set_lv, only: [:setup_create]
+  before_action :set_cafe_setup, only: [:setup_create]
+
+  def show
+  end
+
+  def new
+  end
+
+  def edit
+  end
+
+  def create
+    flash[:notice] = 'Cafejobbet skapades, success.' if @cwork.save
+    respond_with @cwork
+  end
+
+  def update
+    if @cwork.update(c_w_params)
+      flash[:notice] = 'Cafejobbet uppdaterades'
+      redirect_to @cwork
+    else
+      render action: :edit
+    end
+  end
+
+  def destroy
+    # Id used to hide element
+    @id = @cwork.id
+    @cwork.destroy
+    respond_to do |format|
+      format.html { redirect_to :hilbert, notice: 'Cafepasset raderades.' }
+      format.js
+    end
+  end
+
+  def remove_worker
+    if !@cwork.clear_worker
+      render action: show, notice: "Lyckades inte"
+    end
+  end
+
+  def setup
+  end
+
+  def setup_create
+    if preview?
+      @cworks = @r.preview(@lv_first, @lv_last)
+    elsif save?
+      @r.setup(@lv_first, @lv_last)
+      flash[:notice] = "Skapades"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.